src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -240,7 +239,7 @@ npm install aws-amplify
 ```
 
 In your app's entry point (i.e. **App.js**, **index.js**, **_app.js**, or **main.js**), import and load the configuration file:
-
+ 
 <Callout>
 
 **Note:** If you used the "Existing Resource" tab above, you can skip this step.
```

Comment:
Hmmm - seeing this - it would probably make more sense to then include the steps on configuring the library within the "path" you select for your resource set up. Either ways it is super confusing to mention this again.